libs/sqlite3/Makefile

```diff
@@ -130,7 +132,7 @@ endef
 
 define Package/libsqlite3/install
 	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libsqlite3.so.$(ABI_VERSION)* $(1)/usr/lib
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libsqlite3.so* $(1)/usr/lib
```

Comment:
Was this change intentional? I couldnt see it described anywhere.

Reply:
Yes. The sonames are different by default: the so.0 is still there, but the rest are so.3.something now, which so.0 links to.